lib/ssl/src/dtls_packet_demux.erl

```diff
@@ -297,6 +297,18 @@ do_set_emulated_opts([], Opts) ->
     Opts;
 do_set_emulated_opts([{mode, Value} | Rest], Opts) ->
     do_set_emulated_opts(Rest,  Opts#socket_options{mode = Value}); 
+do_set_emulated_opts([{active, N0} | Rest], Opts=#socket_options{active = Active}) when is_integer(N0) ->
```

Comment:
I noticed this construction several times. Is it possible to combine into one (utility) function?

/BMK

Reply:
I have added a common function for this.